ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecryptJWE.java

```diff
@@ -0,0 +1,180 @@
+package com.mozilla.telemetry.decoder;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.mozilla.telemetry.ingestion.core.Constant.Attribute;
+import com.mozilla.telemetry.ingestion.core.schema.PipelineMetadataStore;
+import com.mozilla.telemetry.ingestion.core.schema.PipelineMetadataStore.JweMapping;
+import com.mozilla.telemetry.ingestion.core.schema.PipelineMetadataStore.PipelineMetadata;
+import com.mozilla.telemetry.transforms.FailureMessage;
+import com.mozilla.telemetry.transforms.PubsubConstraints;
+import com.mozilla.telemetry.util.BeamFileInputStream;
+import com.mozilla.telemetry.util.GzipUtil;
+import com.mozilla.telemetry.util.Json;
+import com.mozilla.telemetry.util.KeyStore;
+import com.mozilla.telemetry.util.KeyStore.KeyNotFoundException;
+import java.io.IOException;
+import java.security.PrivateKey;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.beam.sdk.io.gcp.pubsub.PubsubMessage;
+import org.apache.beam.sdk.options.ValueProvider;
+import org.apache.beam.sdk.transforms.FlatMapElements;
+import org.apache.beam.sdk.transforms.PTransform;
+import org.apache.beam.sdk.transforms.ProcessFunction;
+import org.apache.beam.sdk.transforms.WithFailures;
+import org.apache.beam.sdk.transforms.WithFailures.Result;
+import org.apache.beam.sdk.values.PCollection;
+import org.apache.beam.sdk.values.TypeDescriptor;
+import org.jose4j.jwe.JsonWebEncryption;
+import org.jose4j.lang.JoseException;
+
+public class DecryptJWE extends
```

Comment:
```suggestion
public class DecryptJwe extends
```

I have a preference to stick with using only first letter capitalized for initialisms used in camelcase contexts. We already do that in this codebase for `JweMapping`, for example.

Reply:
It's an actual linting error caught by CI, so this will get changed. I'll be renaming the whole module to include more business logic alongside the decryption behavior anyhow.